### PR TITLE
Rust 1.73

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: shotover-proxy/Cargo.toml [hunk 1]
```diff
@@ -49,7 +49,7 @@ rustls-pemfile = "1.0.2"
 aws-throwaway.workspace = true
 windsock = { path = "../windsock" }
 regex = "1.7.0"
-cql-ws = { git = "https://github.com/conorbros/cql-ws" }
+cql-ws = { git = "https://github.com/shotover/cql-ws" }
```
Comment (rukai) [inline shotover-proxy/Cargo.toml:52]: oops

Comment (conorbros) [reply shotover-proxy/Cargo.toml:52]: Yeah, I'll just merge the other one first

Comment (rukai) [reply shotover-proxy/Cargo.toml:52]: would be good to fix it here, to avoid the unnecessary dependency.

**Review by rukai — APPROVED** [review]
Thanks for doing this!

**Review by cjrolo — APPROVED** [review]

**Pull request merged**